astropy/config/configuration.py

```diff
@@ -98,6 +98,25 @@ class Conf(_config.ConfigNamespace):
                 aliases=['astropy.utils.console.USE_COLOR'])
         conf = Conf()
     """
+    def __iter__(self):
+        for key, val in self.__class__.__dict__.items():
+            if isinstance(val, ConfigItem):
+                yield key
+
+    keys = __iter__
+    """Iterate over configuration item names."""
+
+    def values(self):
+        """Iterate over configuration item values."""
+        for key, val in self.__class__.__dict__.items():
```

Comment:
I'd iterate over `__dict__.values()` here, but not essential.

Reply:
Good catch!